### PR TITLE
feat(NoPush): Added: no pushes from more of anything.

### PR DESCRIPTION
**Pull request by sqlerrorthing**
Pushing off: 
* Entites (players, and others)
* Blocks (not kicked out of the blocks)
* Liquids (no water/lava shocks)
* Sinking (don't go down in the water/lava)

### CODE REVIEW

File: src/main/kotlin/net/ccbluex/liquidbounce/utils/input/InputUtil.kt [hunk 1]
```diff
@@ -58,30 +60,33 @@ fun inputByName(name: String): InputUtil.Key {
...
-        .map { key -> reduceInputName(key.translationKey) }
-        .toSet()
+    get() =
+        InputUtil.Type.KEYSYM.map.values
+            .map { key -> reduceInputName(key.translationKey) }
+            .toSet()
 
 fun ActionResult.shouldSwingHand() = this is ActionResult.Success && this.swingSource == ActionResult.SwingSource.CLIENT
+
+operator fun KeyBinding.invoke(): Boolean = this.isPressed
```
Comment (1zun4) [inline src/main/kotlin/net/ccbluex/liquidbounce/utils/input/InputUtil.kt]: That might lead to confusion.

File: src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/movement/ModuleNoPush.kt [hunk 2]
```diff
@@ -18,12 +18,45 @@
...
+
+    fun isEntities() = running && entities
+
+    fun isBlocks() = running && blocks
+
+    fun isFishingRoads() = running && fishingRoads
+
+    @Suppress("unused")
+    private val tickHandler =
+        handler<GameTickEvent> {
```
Comment (1zun4) [inline src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/movement/ModuleNoPush.kt:48]: linebreak?

Comment (sqlerrorthing) [reply src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/movement/ModuleNoPush.kt:48]: ktlint

File: src/main/kotlin/net/ccbluex/liquidbounce/utils/input/InputUtil.kt (hunk 1 above)
Comment (sqlerrorthing) [reply src/main/kotlin/net/ccbluex/liquidbounce/utils/input/InputUtil.kt]: you're suggesting we take it out?

Comment (1zun4) [reply src/main/kotlin/net/ccbluex/liquidbounce/utils/input/InputUtil.kt]: Yes. I think so.

Comment (sqlerrorthing) [reply src/main/kotlin/net/ccbluex/liquidbounce/utils/input/InputUtil.kt]: alr

**Pull request merged**